src/fare-contracts/utils.ts

```diff
@@ -120,22 +120,14 @@ export const useNonInspectableTokenWarning = (fareProductType?: string) => {
       return t(FareContractTexts.warning.errorWithToken);
     case 'other':
       const inspectableToken = tokens.find((t) => t.isInspectable);
-      if (fareProductType === 'carnet') {
```

Comment:
This is the tricky part fareProductType === 'carnet' was removed, but that removes the two different messages, so I think is fine then.

Reply:
It is the tricky part, yes 😄

In practice, it means we treat carnets like other tickets now, since they are also inspectable on phone: We simply message about token being set to another device if that is the case.